lib/src/core/common/widgets/customized_appbar.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+import 'package:window_manager/window_manager.dart';
+
+class CustomAppBar extends StatelessWidget implements PreferredSizeWidget {
+  const CustomAppBar({super.key});
+
+  @override
+  Size get preferredSize => const Size.fromHeight(48);
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onPanStart: (details) async {
+        await windowManager.startDragging();
+      },
+      child: Container(
+        height: preferredSize.height,
+        padding: const EdgeInsets.symmetric(horizontal: 10),
+        decoration: BoxDecoration(
+          color: Colors.grey[900],
```

Comment:
The application supports light & dark mode. Please remove the hardcode of the background.

Reply:
@phoenixit99  / @PouriaMoradi021 

These colors should be defined and managed within a color palette. 
The application should support switching between light and dark themes.

**Dependencies:**

* This feature depends on the resolution of Pull Request #45.
* Access to the Figma design system is required.